### DISCUSSION TITLE
Why does the ISO require 99GB to install

### DISCUSSION THREAD

**Discussion by B3DTech** [Security-Onion-Solutions/securityonion#1637] [General] [answered]
I'm really just trying to get an analyst station stood up.  Why does it require 99GB minimum disk space to even start the install process, especially for an analyst station that wouldn't be storing large amounts of data?

**Comment by TOoSmOotH** [comment, marked as answer]
The ISO install is tailored to setting up something that will have docker containers. You can do a centos minimal install and clone the repo and run the analyst install from there. 

We plan on adding an airgap option in the near future that will be able to build from ISO.

**Reply by B3DTech** [reply]
It really seems like the installer is done completely backwards.  It should prompt you what type of install you want to do first, then verify resources, then do the install.

**Reply by TOoSmOotH** [reply]
The reason it seems backwards it because it was. The analyst workstation sprint came towards the end of the SO2 dev cycle. The ISO install will continue to see improvements, but we added the network install option so that folks could customize their own disk layouts as they see fit. We will have an upcoming sprint that will improve the analyst workstation install experience and allow it to work in an airgap scenario.
